src/functionalTest/resources/application-integration.yml

```diff
@@ -32,6 +32,7 @@ archival-date-format: dd-MM-yyyy
 active-blob-path: azure-blob://${azure.storage.account-name}/jud-ref-data
 file-read-time-out: 2000
 scheduler-audit-select: select count(*) from dataload_schedular_audit where date(scheduler_start_time) = current_date
+scheduler-audit-prev-day: select count(*) from dataload_schedular_audit where date(scheduler_start_time) = current_date - INTEGER '1'
```

Comment:
I think this property (scheduler-audit-prev-day) is required in application-camel-routes-common.yaml  file as well.